thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/authorization/EntityType.java

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright 2022 StarTree Inc
+ *
+ * Licensed under the StarTree Community License (the "License"); you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at http://www.startree.ai/legal/startree-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT * WARRANTIES OF ANY KIND,
+ * either express or implied.
+ * See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package ai.startree.thirdeye.spi.authorization;
+
+public enum EntityType {
```

Comment:
What is this entityType for? We already have different entity types in the persistence layer

Reply:
This is used for the startree resource name https://github.com/startreedata/startree-pinot/blob/master/startree-plugins/startree-auth-platform/src/main/java/ai/startree/pinot/auth/platform/SRN.java

I _kinda_ think it makes sense to use a separate entity type enum for the access control entity types, but as long as I can get a string entity name, that's all that matters.

Reply:
Is this what you had in mind? https://github.com/startreedata/thirdeye/pull/803/files#diff-eda7b7b48af5e20539ea96eab104586de9e4fc73adff1b44d0026b6d975338c6R20